src/Features/Core/Portable/FindUsages/DefinitionItem.cs

```diff
@@ -154,11 +158,29 @@ internal abstract partial class DefinitionItem
                 Contract.ThrowIfFalse(Properties.ContainsKey(MetadataSymbolOriginatingProjectIdGuid));
                 Contract.ThrowIfFalse(Properties.ContainsKey(MetadataSymbolOriginatingProjectIdDebugName));
             }
+
+            Workspace = sourceSpans.FirstOrDefault().Document?.Project.Solution.Workspace;
```

Comment:
💭 This seems concerning. If we need the workspace, it seems we should pass it explicitly.

Reply:
I could do that.